extensions/wikia/LookupUser/templates/contribution.table.tmpl.php

```diff
@@ -129,15 +129,19 @@
 					//changing placeholders with ajax loading gifs
 					$('.user-groups-placeholder').each(function(){
 						var self = $(this);
-						var wikiId = self.find('input.wikiId').val();
+						var wikiId = parseInt( self.find('input.wikiId').val() );
 						var url = self.find('input.wikiUrl').val();
-						var userName = self.find('input.name').val();
+						var username = self.find('input.name').val();
 
 						try {
 							var ajaxRequest = $.ajax({
 								dataType: 'json',
 								type: "POST",
-								data: 'url=' + url + '&username=' + username + '&id=' + wikiId,
+								data: {
+									url: mw.html.escape( url ),
+									username: mw.html.escape( username ),
```

Comment:
Escaping here is unneeded as it's just POST data (and could actually lead to incorrect data if the username has a HTML entity in it.